ccd-definition/AuthorisationCaseField/CareSupervision/gatekeeper.json

```diff
@@ -2274,6 +2274,48 @@
     "UserRole": "caseworker-publiclaw-gatekeeper",
     "CRUD": "CRU"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "reviewCMOOthersHeader",
+    "UserRole": "caseworker-publiclaw-gatekeeper",
+    "CRUD": "CRU"
```

Comment:
Only a header so maybe only needs R

Reply:
these fields no longer exist. I've updated the code to reuse otherDetailsSectionSubHeader, otherDetailsSectionHeader, sendOrderToAllOthers fields